pkgs/test/test/runner/configuration/runner_output_test.dart

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) 2020, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'dart:io';
+
+@TestOn('vm')
```

Comment:
This needs to be the first non-comment line in the file, above `import 'dart:io';`.

Reply:
I pushed a new version, with the @TestOn on the top